lib/utils/util.js

```diff
@@ -0,0 +1,59 @@
+/** @flow */
+
+import {spawn} from 'child_process';
+import {window, workspace} from 'vscode'
+
+const NODE_NOT_FOUND = '[Flow] Cannot find node in PATH. The simpliest way to resolve it is install node globally'
+const FLOW_NOT_FOUND = '[Flow] Cannot find flow in PATH. Try to install it by npm install flow-bin -g'
+
+function getPathToFlow(): string {
+	return workspace.getConfiguration('flow').get('pathToFlow')
+}
+
+export function isFlowEnabled() {
+	return workspace.getConfiguration('flow').get('enabled')
+}
+
+export function checkNode() {
```

Comment:
No, we don't want to ship `flow-bin` with this package because that locks the editor plugin to a specific version of Flow. Flow is rapidly changing and each version introduces breaking changes, so we definitely don't want to force people to upgrade their Flow versions with their editor.

We also shouldn't reach in to individual projects' `flow-bin` installation, because it's a security risk. Some discussion on that here: https://github.com/facebook/nuclide/issues/570